Source/SwiftLintFramework/Rules/Metrics/EnumCaseAssociatedValuesLengthRule.swift

```diff
@@ -0,0 +1,81 @@
+import SourceKittenFramework
+
+public struct EnumCaseAssociatedValuesLengthRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityLevelsConfiguration(warning: 5, error: 6)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "enum_case_associated_values_count",
+        name: "Enum Case Associated Values Count",
+        description: "Enum case's associated value length should be low",
```

Comment:
can you replace "length" with "count" or use something like "Number of associated values in an enum case should be low."